client/my-sites/stats/stats-subscribers-overview/index.tsx

```diff
@@ -1,57 +1,113 @@
 import { CountComparisonCard } from '@automattic/components';
+import { UseQueryResult, useQueries } from '@tanstack/react-query';
 import { translate } from 'i18n-calypso';
 import React from 'react';
+import {
+	querySubscribers,
+	selectSubscribers,
+} from 'calypso/my-sites/stats/hooks/use-subscribers-query';
+
+const indexFirstCard = 0; // 0 for today
+const indexSecondCard = 30; // 30 days out
+const indexThirdCard = 60; // 60 days out
+const indexFourthCard = 90; // 90 days out
+
+interface SubscribersData {
+	period: string;
+	subscribers: number;
+	subscribers_change: number;
+}
+
+interface SubscribersDataResult {
+	data: SubscribersData[];
+	unit: string;
+	date: string;
+}
+
+interface SubscribersOverviewProps {
+	siteId: number;
+}
+
+// calculate the dates to query for
+const dateToday = calculateQueryDate( indexFirstCard );
+const date30DaysAgo = calculateQueryDate( indexSecondCard );
+const date60DaysAgo = calculateQueryDate( indexThirdCard );
+const date90DaysAgo = calculateQueryDate( indexFourthCard );
+
+// calculate the date to query for based on the number of days to subtract
+function calculateQueryDate( daysToSubtract: number ) {
+	const today = new Date();
+	const date = new Date( today );
+	date.setDate( date.getDate() - daysToSubtract );
+	return date.toISOString().split( 'T' )[ 0 ];
+}
+
+// calculate the stats to display in the cards
+function SubscribersOverviewCardStats( subscribersData: SubscribersData[][] ) {
```

Comment:
I would prefer a different name for `subscribersData` as it's misleading - only when `subscribersData` is of type `SubscribersData`, we'd use the name IMO. `subscribersData` here is actually a two dimensional array of SubscribersData. I would use something like `subscribersDataArrays` but I'm not very sure to be honest...